src/Pages/CreatorPage.js

```diff
@@ -1,8 +1,8 @@
 import React, { useEffect, useState } from 'react';
 import { useParams } from 'react-router-dom';
 // import propTypes from 'prop-types';
-import AuthorInfo from '../Components/CreatorPage/AuthorInfo';
-import CreatedWorksCard from '../Components/CreatorPage/OtherWorks';
+import AuthorInfo from '../Components/creatorPage/AuthorInfo';
```

Comment:
Main branch should have all of the component folders capitalized, can you change that locally?